packages/replay-internal/src/types/performance.ts

```diff
@@ -111,6 +111,10 @@ export interface WebVitalData {
    * The recording id of the web vital nodes. -1 if not found
    */
   nodeIds?: number[];
+  /**
+   * The layout shifts of a CLS metric
+   */
+  layoutShift?: { value: number; sources?: number[] }[];
```

Comment:
I would give this a more generic name, otherwise we have a 'WebVitalData' type that's generic with a `layoutShift` property that is very specific to a type of metric.

Maybe: `attributions?: {value: number; nodeIds?: number}[]`? or `sources?: {value: number; nodeIds?: number}[]`

Reply:
good point!